### DISCUSSION TITLE
动漫资源的检索问题

### DISCUSSION THREAD

**Discussion by dlovew** [jxxghp/MoviePilot#373] [讨论]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

1.0.9

### 问题类型

主程序运行问题

### 问题描述

搜索动漫时出现无法检索到对应资源的问题，部分动漫比如观众的一部动漫
标题：Suki na Ko ga Megane wo Wasureta S01E09 2023 1080p Baha WEB-DL x264 AAC-AnimeS@ADWeb
副标题：2023年7月新番 喜欢的女孩忘戴眼镜了 / 好きな子がめがねを忘れた 第09集
可以正常识别，但是在检索时却检索不到
初步怀疑是副标题的问题，tmdb里面动漫的中文名为：喜欢的人忘记戴眼镜了
类似的动漫有不少，不知道能不能解决，先提交了

**Comment by xyswordzoro** [comment]
原因很简单，就是因为名字不一样，你用观众搜 喜欢的人忘记戴眼镜了  也搜不出官组的资源
  官组的命名是：喜欢的女孩忘戴眼镜了
